src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -1672,10 +1654,11 @@ void RequireDynamicallyAccessedMembers (ref ReflectionPatternContext reflectionC
 			reflectionContext.RecordHandledPattern ();
 		}
 
-		static bool BindingFlagsAreSupported (BindingFlags bindingFlags)
-		{
-			return (bindingFlags & BindingFlags.IgnoreCase) == BindingFlags.IgnoreCase || (int) bindingFlags > 255;
-		}
+		static BindingFlags? GetBindingFlags (ValueNode parameter) => (BindingFlags?) parameter.AsConstInt ();
```

Comment:
Nit: I would rename this to `GetBindingFlagsFromValue` - just to make a bit more explicit.